nix/pkgs/thorp/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, sources }:
```

Comment:
As mentioned above it would be nice if you could change this expression to receive the dependencies that actually need to be injected instead of passing on the whole `pkgs`:

```
{ stdenv, makeWrapper, jdk11_headless, maven, graphviz, src }: 
```
something like that would do I think..